### ISSUE TITLE
Duplicated language strings (7.10-beta-2)

### ISSUE THREAD

**Issue by likhobory** [salesagility/SuiteCRM#4637]
'LBL_LOGOUT'
https://github.com/salesagility/SuiteCRM/blob/v7.10-beta-2/include/language/en_us.lang.php#L1058
https://github.com/salesagility/SuiteCRM/blob/v7.10-beta-2/include/language/en_us.lang.php#L1700

'ERR_PASSWORD_ONEUPPER'
https://github.com/salesagility/SuiteCRM/blob/v7.10-beta-2/modules/Users/language/en_us.lang.php#L75-L76

It should be 'ERR_PASSWORD_**ONELOWER**' on line 76.

**Comment by horus68** [comment]
`ERR_PASSWORD_ONEUPPER` isn't just a duplicated, its a bug! @Dillon-Brown 
```
    'ERR_PASSWORD_ONEUPPER' => 'Password should contains uppercase chars.',
    'ERR_PASSWORD_ONEUPPER' => 'Password should contains lowercase chars.',
```

**Comment by Mausino** [comment]
@horus68 @likhobory you're my language heros  :) Your great work spread SuiteCRM to all world for many different nations ;)

**Comment by Mausino** [comment]
@horus68 @likhobory you're my language heroes  :) Your great work spreads SuiteCRM to all world for many nations ;)

**Comment by Dillon-Brown** [comment]
Fixed the ERR_PASSWORD_ONEUPPER issue in #4637 and the LBL_LOGOUT duplication in #4839. Thanks!
